docs/source/examples/syncing-code-artifacts.rst

```diff
@@ -49,8 +49,8 @@ scripts, access checkpoints, etc.).
 
     For large, multi-gigabyte workdirs, uploading may be slow because they
     are synced to the remote VM(s) with :code:`rsync`. To exclude large files in
-    your workdir from being uploaded, add them to the :code:`.gitignore` file
-    (or a ``.git/info/exclude`` file) under the workdir.
+    your workdir from being uploaded, add them to a :code:`.skyignore` file 
+    under your workdir.
```

Comment:
mention the syntax, e.g., ``the syntax should be compatible with `rsync` filter rules``, and add an example below:
```
# Exclude all log files in every subdir
*.log
# Exclude .txt file under ./myfolder
/myfolder/*.txt
# Exclude wandb folder
wandb/
```